lib/src/main/java/org/openapitools/openapistylevalidator/ErrorAggregator.java

```diff
@@ -86,4 +86,8 @@ public void logMissingPathsAndComponents() {
                 "paths,components",
                 "Should have at least one of paths or components"));
     }
+
+    public void logUndefinedTag(String missingTagName) {
```

Comment:
This is too "precise" for this class. If you check the other log, they are very generic.

Reply:
yes.. I want to get my design to work and get early feedback.. I should refactor this to be more generic. I will fix it.

Reply:
thanks!